UI/DevScreens.cpp

```diff
@@ -400,13 +400,13 @@ void SystemInfoScreen::CreateViews() {
 #endif
 #endif
 	if (GetGPUBackend() == GPUBackend::OPENGL) {
-		deviceSpecs->Add(new InfoItem(si->T("Core Context"), gl_extensions.IsCoreContext ? si->T("Yes") : si->T("No")));
+		deviceSpecs->Add(new InfoItem(si->T("Core Context"), gl_extensions.IsCoreContext ? di->T("Supported") : di->T("Unsupported")));
```

Comment:
Hmm, maybe "Enabled" or "Active"?  It could be supported but off.

Actually either of those would work for RW/RX too.  Debugger isn't very important since only developers will see it.

-[Unknown]

Reply:
Thanks for your suggestions :)